contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -545,6 +545,29 @@ namespace eosiosystem {
       asset stake_change;
    };
 
+   struct action_return_sellram {
+      name account;
+      asset quantity;
+      int64_t bytes_sold;
+      int64_t ram_bytes;
+   };
+
+   struct action_return_buyram {
+      name payer;
+      name receiver;
+      asset quantity;
+      int64_t bytes_purchased;
+      int64_t ram_bytes;
+   };
+
+   struct action_return_ramtransfer {
+      name from;
+      name to;
+      int64_t bytes;
```

Comment:
`bytes` could be renamed to `transferred_bytes` in the Reference Contracts. Since this PR is a merge. Don't change it now.

Reply:
`ramburn` also returns the `action_return_ramtransfer`, and for that reason we left the field name as `bytes`.